Makefile

```diff
@@ -41,8 +41,9 @@ unlock: pristine ## Unlock dependencies
 	touch package.json
 	touch yarn.lock
 
-pristine: clean ## Remove all `node_modules` and files
-	rm -rf node_modules {packages,apps}/*/node_modules
+pristine: clean ## Remove all `node_modules`, `vendor` and files
+	-rm -rf node_modules {packages,apps}/*/node_modules
```

Comment:
it doesn't remove `node_modules` in `/exporters/variables-scss/` and `/examples/...` folders

Reply:
Great! Thanks for noticing 👍 Fixed.